docs/sources/configuration/_index.md

```diff
@@ -2372,6 +2372,10 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # This also determines how cache keys are chosen when result caching is enabled
 # CLI flag: -querier.split-queries-by-interval
 [split_queries_by_interval: <duration> | default = 30m]
+
+# When true, access to the deletion API on the compactor is enabled.
+# CLI flag: -compactor.deletion-enabled
+[compactor_deletion_enabled: <boolean> | default = false]
```

Comment:
Is this config a bit too specific with implementation details? Could this not be renamed to `allow_deletes`?